lib/plug/adapters/cowboy/conn.ex

```diff
@@ -31,6 +31,7 @@ defmodule Plug.Adapters.Cowboy.Conn do
   end
 
   def send_resp(req, status, headers, body) do
+    status = "#{status} #{Plug.Conn.Status.string(status)}"
```

Comment:
Let's wait until more than one place needs it. Although I would write the line above as:

    status = Integer.to_string(status) <> " " <> Plug.Conn.Status.reason_phrase(status)